tests/test_parsing.py

```diff
@@ -83,3 +84,15 @@ def test_boolean_param_parse(boolean_param_config):
 def test_mount_parse(mount_config):
     step = mount_config.steps['test']
     assert len(step.mounts) == 2
+
+
+def test_endpoint_parse(endpoint_config):
+    server_endpoint = endpoint_config.endpoints['server-endpoint']
+    assert server_endpoint.image == 'python:3.6'
+    assert server_endpoint.port == 1453
+    assert server_endpoint.server_command == 'python run_server.py'
+    wsgi_endpoint = endpoint_config.endpoints['wsgi-endpoint']
+    assert wsgi_endpoint.description == 'predict digits from image inputs'
+    assert wsgi_endpoint.image == 'gcr.io/tensorflow/tensorflow:1.3.0-py3'
+    assert wsgi_endpoint.wsgi == 'predict_wsgi:predict_wsgi'
+    assert len(wsgi_endpoint.files) == 1
```

Comment:
Also, mind adding a test for serialization/roundtripping of endpoint data?

Reply:
The tests are already doing this, they are running each test twice, once with and once without doing the roundtripping.